src/components/GrampsjsMapTimeSlider.js

```diff
@@ -0,0 +1,58 @@
+import {html, css, LitElement} from 'lit'
+import '@material/web/slider/slider.js'
+
+import {sharedStyles} from '../SharedStyles.js'
+import {GrampsjsTranslateMixin} from '../mixins/GrampsjsTranslateMixin.js'
+import {fireEvent} from '../util.js'
+
+class GrampsjsMapTimeSlider extends GrampsjsTranslateMixin(LitElement) {
+  static get styles() {
+    return [
+      sharedStyles,
+      css`
+        #container {
+          z-index: 999;
+          background-color: #ffffff;
+          border-radius: 14px;
+          width: calc(100% - 150px);
+          position: absolute;
+          bottom: 25px;
+          height: 28px;
+          left: 50%;
+          transform: translateX(-50%);
+          display: flex;
+          justify-content: center;
+          align-items: center;
+        }
+
+        md-slider {
+          width: 100%;
+        }
+      `,
+    ]
+  }
+
+  render() {
+    return html`
+      <div id="container">
+        <md-slider
+          @change="${this._handleInput}"
```

Comment:
Would it be possible to fire this event continuously as the user scrubs the slider? The user might find it confusing at first if the map doesn’t seem to update until they let go of the knob.

Reply:
True. We can just change it to `@input`.